middleware.ts

```diff
@@ -47,8 +47,8 @@ export async function middleware(req: NextRequest) {
 
   if (
     !pathsToMatch.some((matcher) => pathToRegexp(matcher).test(req.nextUrl.pathname)) ||
-    // if the path is hub/insights go to the page so logged out users can see demo insights
-    req.nextUrl.pathname === "/hub/insights"
+    // if the path is hub/insights or hub/lists go to the page so logged out users can see demo insights or demo lists    req.nextUrl.pathname === "/hub/insights" ||
```

Comment:
> This part (req.nextUrl.pathname === "/hub/insights" ||) is actually included in the comment

Thanks for catching this @ThePiyushAggarwal. looks kinda tricky to detect at first glance :<


@sourabpramanik please remove check condition from the commented code